v0.21.0 fails to build on FreeBSD

I'm seeing a build failure on FreeBSD:

```
github.com/zaquestion/lab/cmd
# github.com/zaquestion/lab/cmd
cmd/util_unix.go:33:9: undefined: syscall.Dup3
```
Yes, the dup3 syscall exists, but I think you can't access it from go via `syscall`. I think you have to use `x/sys/unix` Syscall()

--- a/cmd/util_bsd.go
+++ b/cmd/util_bsd.go
@@ -1,6 +1,6 @@
 // This file contains Darwin (MacOS) and *BSD specific calls.
 
-// +build darwin
+// +build freebsd openbsd dragonfly darwin
 
 package cmd
 

--- a/cmd/util_bsd.go
+++ b/cmd/util_bsd.go
@@ -1,4 +1,4 @@
-// This file contains Darwin (MacOS) specific calls.
+// This file contains Darwin (MacOS) and *BSD specific calls.
 
 // +build darwin
 
@@ -8,8 +8,8 @@ package cmd
 // by Linux ARM64 not having the DUP2() anymore. With that, we need to
 // repeat the other code and func declarations that are the same.
 
-// FIXME: there MUST be some better way to do that... only dupFD2() should be
-// here.
+// FIXME: there MUST be some better way to do that... only dupFD2() should
+// be here.
 
 import "syscall"
 

--- a/cmd/util_linux.go
+++ b/cmd/util_linux.go
@@ -1,13 +1,10 @@
 // This file contains Linux specific calls.
 
-// +build !windows,!darwin
-
 package cmd
 
 // Since we're using some system calls that are platform-specific, we need
-// to make sure we have a small layer of compatibility for Unix-like and
-// Windows operating systems. For now, this file is still valid for BSDs
-// (MacOS NOT included)
+// to make sure we have a small layer of compatibility for Linux, Windows
+// and *BSD operating systems.
 
 import "syscall"
 

--- a/cmd/util_windows.go
+++ b/cmd/util_windows.go
@@ -1,7 +1,5 @@
 // This file contains Windows specific calls.
 
-// +build windows
-
 package cmd
 
 // Even though Windows has a POSIX layer, it's implemented in userspace and,

